addons/account/models/ir_actions_report.py

```diff
@@ -45,3 +48,11 @@ def render_qweb_pdf(self, res_ids=None, data=None):
                     raise UserError(_("Only invoices could be printed."))
 
         return super().render_qweb_pdf(res_ids=res_ids, data=data)
+
+    def _get_rendering_context(self, docids, data):
+        data = data and dict(data) or {}
+        ids = isinstance(docids, int) and [docids] or docids
+        invoice = self.env['account.move'].search([('id', 'in', ids)])
+        rounding = invoice and min(invoice.mapped('currency_id.rounding')) or 0.0
```

Comment:
```python
min(invoice.mapped('currency_id.rounding'), default=0.0)
```

But this doesn't make any sense anyway. You should take the correct rounding for each individual invoice when looping in the `t-foreach`

Reply:
My thought was : as the bug was happening because of the way floats are managed by python (178.2000000000002), the best accurate method is to compare with the minimum rounding. In any case if the rounding processed before is done correctly, the float_compare will return 0, don't you think?

Reply:
Not with the minimum rounding, but with the rounding of the currency set on the invoice.

Reply:
oh, ok I see what you meant, I was getting so confused, sorry